pkg/chunk/azure/blob_storage_client_test.go

```diff
@@ -0,0 +1,94 @@
+package azure
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"io/ioutil"
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/cortexproject/cortex/pkg/util/flagext"
+)
+
+func TestGetObject(t *testing.T) {
```

Comment:
I tried to run these tests locally, but looks failing with the error `illegal base64 data at input byte 8`. Also I'm not sure how they're supposed to work, considering the backend storage is not mocked 🤔

Reply:
I think you need a account key of storageaccount, like ”gN+OpbBAYqXHvLSJQhMaMaUBHuViamwrongkey0nGUuBXfWPhIO0xhH3iFzg==“

Azure encrypts the accountkey, so we can’t use "accountkey" directly

Reply:
Yes, I think it is difficult for me to mock azure storage, so I deleted the test file 😂